Makefile

```diff
@@ -249,3 +249,15 @@ release-manifests: $(KUSTOMIZE) ## Create kustomized release manifest in $RELEAS
 release-templates: ## Generate release templates
 	@mkdir -p $(RELEASE_DIR)
 	cp templates/cluster-template*.yaml $(RELEASE_DIR)/
+
+
+##@ CRS
+## --------------------------------------
+## CRS
+## --------------------------------------
+
+CALICO_VERSION ?= v3.26.3
```

Comment:
3.28.0 is already out. can we choose a new version?

Reply:
ok, what about 3.27.x?

Reply:
Yeah, the annotation value is not feasible for some things. Kamaji has the same problem. Maybe something like `kubectl create || kubectl replace` could work?

Reply:
I don't see a point to discuss in which way we would create the configmap, 
as it depends on the setup.

The Kubectl create command is just for documentation, informing the user that it must be available in the cluster.
In production environments, the user will choose the solution to create the configmap.